### PR TITLE
FALCON-2061 Falcon CLI shows hadoop classpath loading info in the console

### PR DESCRIPTION
**Pull request by bvellanki**

### CODE REVIEW

**Comment by vrangan** [comment]
+1

File: src/bin/falcon-config.sh [hunk 1]
```diff
@@ -68,18 +68,6 @@ case $type in
     for i in `ls ${BASEDIR}/server/webapp`; do
       FALCONCPPATH="${FALCONCPPATH}:${i}/WEB-INF/lib/*"
     done
-    # In future we should refactor the CP directories between server and client
-    HADOOPDIR=`which hadoop`
-    if [ "$HADOOP_HOME" != "" ]; then
-      echo "Hadoop home is set, adding libraries from '${HADOOP_HOME}/bin/hadoop classpath' into falcon classpath"
-      FALCONCPPATH="${FALCONCPPATH}:`${HADOOP_HOME}/bin/hadoop classpath`"
```
Comment (yzheng-hortonworks) [inline src/bin/falcon-config.sh]: Should the update of FALCONCPPATH be kept here? It is used in later part of the code. Same for the one in "elif".

**Comment by vramachan** [comment]
@bvellanki falcon-config.sh is used by the service-start in addition to falcon cli client. won't the Hadoop libs needed for the server start?

**Comment by vramachan** [comment]
I see you are making the change in the client APPTYPE.
+1

File: src/bin/falcon-config.sh (hunk 1 above)
Comment (bvellanki) [inline src/bin/falcon-config.sh]: The change is only made in Client side. All the jars required by client are in client lib and in server/webapp/web-inf/lib/

**Comment by yzheng-hortonworks** [comment]
+1

**Pull request closed without merging**